console/src/app/pages/actions/add-action-dialog/add-action-dialog.component.ts

```diff
@@ -17,7 +20,7 @@ import { ToastService } from 'src/app/services/toast.service';
   templateUrl: './add-action-dialog.component.html',
   styleUrls: ['./add-action-dialog.component.scss'],
 })
-export class AddActionDialogComponent {
+export class AddActionDialogComponent implements OnInit {
   public name: string = '';
   public script: string = '';
   public durationInSec: number = 10;
```

Comment:
```suggestion
```
not needed anymore if we initialize the form directly and overwrite if anydata is set

Reply:
Code removed!